website/src/pages/log-streaming/content-overview/index.mdx

```diff
@@ -1,46 +1,46 @@
 ---
-Title: Event Streaming
-Description: How does Event Streaming work & how to leverage it
+Title: Log Streaming
+Description: How does Log Streaming work & how to leverage it
 Keywords: 'ibm cloud'
 ---
 
 <PageDescription>
 
-What is Event Streaming?
+What is Log Streaming?
 
 </PageDescription>
 
 
-Event Streaming enables IBM Log Analysis to produce content to a message bus queue and Topic. IBM Log Analysis streaming helps you to connect third party consumers of topics to ingest into dashboards for visualization of event log data. 
+Log Streaming enables IBM Log Analysis to produce content to a message bus queue and Topic. IBM Log Analysis streaming helps you to connect third party consumers of topics to ingest into dashboards for visualization of event log data. 
 
-Third party horizonal technologies such as Splunk, used in organizations for application management, security and compliance are able to leverage IBM Cloud Log Analysis with Event Streaming.
+Third party horizonal technologies such as Splunk, used in organizations for application management, security and compliance are able to leverage IBM Cloud Log Analysis with Log Streaming.
 
-The content in this pattern, we will walk through setting up a sample event stream by utilizing IBM Cloud Event Streams and Event Streaming, to produce events to a topic for a simple consumer application to ingest.
+The content in this pattern, we will walk through setting up a sample event stream by utilizing IBM Cloud Event Streams and Log Streaming, to produce events to a topic for a simple consumer application to ingest.
 
 IBM Cloud Event Streams is a high-through put message bus built with Kafka, see [What is Event Streams](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-about) and [Choosing Your Plan](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-plan_choose)
```

Comment:
While we are updating this content, pls replace all cases of "high-through put" with the correct grammar: "high throughput" .